### PR TITLE
feat: Add user total holding input in Mana Calculator

### PR DESCRIPTION
**Pull request by marc2332**
Depends on https://github.com/iotaledger/iota-wiki/pull/1314
Closes https://github.com/iotaledger/iota-wiki/issues/1269

### CODE REVIEW

**Review by begonaalvarezd — COMMENTED** [review]
👯

**Review by begonaalvarezd — APPROVED** [review]

File: src/components/ManaCalculator/components/RoleSection.tsx [hunk 1]
```diff
@@ -33,12 +34,19 @@ export function RoleSection() {
         ]}
       />
       <br />
+      <label className='inlined-label'>Holded amount</label>
```
Comment (Dr-Electron) [inline src/components/ManaCalculator/components/RoleSection.tsx:37]: I think it is `Held`, no?
```suggestion
      <label className='inlined-label'>Held amount</label>
```

Comment (Dr-Electron) [reply src/components/ManaCalculator/components/RoleSection.tsx:37]: If I'm correct, you can probably change the var names too

Comment (marc2332) [reply src/components/ManaCalculator/components/RoleSection.tsx:37]: oooooops

**Pull request merged**